Procurement/Controls/ItemHover.xaml

```diff
@@ -423,17 +432,40 @@
                     <TextBlock Width="{Binding Path=ActualWidth,
                                                ElementName=ItemHeader,
                                                Mode=OneWay}"
-                               HorizontalAlignment="Center"
-                               FontFamily="../Resources/#Fontin SmallCaps"
-                               Foreground="#960003"
-                               Padding="10 0 10 5"
-                               Text="Corrupted"
-                               TextAlignment="Center"
-                               TextWrapping="Wrap"
-                               Visibility="{Binding IsCorrupted,
+                        HorizontalAlignment="Center"
+                        FontFamily="../Resources/#Fontin SmallCaps"
+                        Foreground="#960003"
+                        Padding="10 0 10 5"
+                        Text="Corrupted"
+                        TextAlignment="Center"
+                        TextWrapping="Wrap"
+                        Visibility="{Binding IsCorrupted,
                                                     Converter={StaticResource bc},
                                                     ConverterParameter=CollapseWhenFalse}" />
 
+                    <StackPanel Orientation="Vertical"
+                                Visibility="{Binding IsGemProgressVisible,
+                                                 Converter={StaticResource VisibilityConverter},
+                                                 ConverterParameter=CollapseWhenFalse}">
+
+                        <DockPanel Margin="10 0 10 5">
+
```

Comment:
Remove the two blank lines here, and the one on line 450.

Reply:
Done